Makefile

```diff
@@ -31,8 +53,33 @@ $(AUTOV8_DEPOT_TOOLS):
 	mkdir -p build
 	cd build; git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
 
+ifeq ($(PLATFORM),arm64.release)
+$(AUTOV8_DIR): $(AUTOV8_DEPOT_TOOLS)
+	# patch with system-installed ninja
+	cp /usr/bin/ninja build/depot_tools/
+
+	# get ARM64 clang binaries
+	cd build; wget -nc http://releases.llvm.org/7.0.1/clang+llvm-7.0.1-aarch64-linux-gnu.tar.xz; tar xf clang+llvm-7.0.1-aarch64-linux-gnu.tar.xz
+
+	# Get and build the plugin
+	cd build; wget -nc https://chromium.googlesource.com/chromium/src/+archive/lkgr/tools/clang/plugins.tar.gz; mkdir -p plugin; cd plugin; tar xf ../plugins.tar.gz; clang++ *.cpp -c -I ../clang+llvm-7.0.1-aarch64-linux-gnu/include/ -fPIC -Wall -std=c++14 -fno-rtti -fno-omit-frame-pointer; clang -shared *.o -o libFindBadConstructs.so
```

Comment:
`clang++` should be changed to `../clang+llvm-7.0.1-aarch64-linux-gnu/bin/clang++`

`clang` should be changed to `../clang+llvm-7.0.1-aarch64-linux-gnu/bin/clang`

or a path needs to be set that includes the directory.

`clang++` also does not appear to be in the path for the actual build of v8, so it looks like you have another dependency installed that isn't documented.

Reply:
The clang package would be a system-level dependency. Once this is installed, clang and clang++ would be in the path.